docs/ldml/tr35-keyboards.md

```diff
@@ -1605,21 +1724,20 @@ _Attribute:_ `from` (required)
 
 - **Fixed character classes and escapes**
 
-    `\s \S \t \r \n \f \v \\ \$`
+    `\s \S \t \r \n \f \v \\ \$ \d \w \D \W` … TODO: check these
```

Comment:
Just flagging this TODO more prominently!

Reply:
```suggestion
    `\s \S \t \r \n \f \v \\ \$ \d \w \D \W \0`
```

Reply:
Hmm, do we need to allow `\0`? Potential for pain there... especially given most implementations are likely to be C-ish and not allow nul bytes in strings. I also can't see it being generally useful in a keyboard